<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizScreen</class>
 <widget class="QWidget" name="QuizScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="quizScrollArea">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="quizContentWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>998</width>
          <height>658</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QPushButton" name="submitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
